packages/coordinator/package.json

```diff
@@ -19,8 +19,9 @@
     "run:node": "node --import 'data:text/javascript,import { register } from \"node:module\"; import { pathToFileURL } from \"node:url\"; register(\"ts-node/esm\", pathToFileURL(\"./\"));'",
     "start": "pnpm run run:node ./ts/main.ts",
     "start:prod": "pnpm run run:node build/ts/main.js",
-    "test": "jest --forceExit",
+    "test": "jest --testPathIgnorePatterns=e2e.aa.test.ts --forceExit",
```

Comment:
I think here we should have unit tests as well and don't be dependant on e2e coverage. It's called test pyramid where we have speed but more isolation and when we go up we have less isolation and tests become slower.